R/plotAbundance.R

```diff
@@ -24,47 +24,55 @@
+#' @param col.var a single \code{character} value defining a column from 
 #'   \code{colData} to be plotted below the abundance plot.
 #'   Continuous numeric values will be plotted as point, whereas factors and
-#'   character will be plotted as colour-code bar. (default: \code{features =
+#'   character will be plotted as colour-code bar. (default: \code{col.var =
 #'   NULL})
 #'   
-#' @param order_rank_by How to order abundance value: By name (\dQuote{name}) 
+#' @param features Deprecated. Use \code{col.var} instead.
+#'   
+#' @param order.row.by How to order abundance value: By name (\dQuote{name}) 
 #' for sorting the taxonomic labels alphabetically, by abundance (\dQuote{abund}) to
 #' sort by abundance values or by a reverse order of abundance values (\dQuote{revabund}).
-#'  
+#' 
+#' @param order_rank_by Deprecated. Use \code{order.row.by} instead.  
 #'   
-#' @param order_sample_by A single character value from the chosen rank of abundance
+#' @param order.col.by A single character value from the chosen rank of abundance
 #'   data or from \code{colData} to select values to order the abundance
-#'   plot by. (default: \code{order_sample_by = NULL})
+#'   plot by. (default: \code{order.col.by = NULL})
 #'   
-#' @param decreasing TRUE or FALSE: If the \code{order_sample_by} is defined and the
+#' @param order_sample_by Deprecated. Use \code{order.col.by} instead.
+#'   
+#' @param decreasing TRUE or FALSE: If the \code{order.col.by} is defined and the
 #'   values are numeric, should the values used to order in decreasing or
 #'   increasing fashion? (default: \code{decreasing = FALSE})
 #'
 #' @param layout Either \dQuote{bar} or \dQuote{point}. 
 #' 
-#' @param one_facet Should the plot be returned in on facet or split into 
+#' @param one.facet Should the plot be returned in on facet or split into 
 #'   different facet, one facet per different value detect in \code{rank}. If
-#'   \code{features} or \code{order_sample_by} is not \code{NULL}, this setting will
+#'   \code{col.var} or \code{order.col.by} is not \code{NULL}, this setting will
 #'   be disregarded.
+#'   
+#' @param one_facet Deprecated. Use \code{one.facet} instead.
 #' 
-#' @param ncol,scales if \code{one_facet = FALSE}, \code{ncol} defines many 
+#' @param ncol,scales if \code{one.facet = FALSE}, \code{ncol} defines many 
 #'   columns should be for plotting the different facets and \code{scales} is
 #'   used to define the behavior of the scales of each facet. Both values are 
 #'   passed onto \code{\link[ggplot2:facet_wrap]{facet_wrap}}.
 #' 
 #' @param ... additional parameters for plotting.
 #'   \itemize{
-#'   \item{use_relative}{ \code{TRUE} or \code{FALSE}: Should the relative values
-#'   be calculated? (default: \code{use_relative = FALSE} }
+#'   \item{as.relative}{ \code{TRUE} or \code{FALSE}: Should the relative values
+#'   be calculated? (default: \code{as.relative = FALSE} }
+#'   \item{use_relative} Deprecated. Use \code{as.relative} instead.
```

Comment:
You can remove use_relative from this documentation. The lack of this internal parameters does not cause checks to fail. Also, if user is using the old code, it is still working even though we are not promoting it here